src/lib/drivers/driver.js

```diff
@@ -159,7 +159,9 @@ class DriverBase {
 
   gotoURL(url, options) {
     const waitForLoad = (options && options.waitForLoad) || false;
+    const disableJavaScript = (options && options.disableJavaScript) || false;
```

Comment:
Eh yeah let's move it above of thirdPass

`// disable JS completely, reload for HTTPS redirect…`